src/nft/components/bag/BagFooter.test.tsx

```diff
@@ -5,13 +5,12 @@ import { useWeb3React } from '@web3-react/core'
 import { nativeOnChain } from 'constants/tokens'
```

Comment:
With the deletion of `src/nft/hooks/usePermit2Approval.test.ts` we do lose the test case that the correct UR address is being used. Can you add that back here?